ps1/ps1/src/twitter/Filter.java

```diff
@@ -1,131 +0,0 @@
-/* Copyright (c) 2007-2016 MIT 6.005 course staff, all rights reserved.
- * Redistribution of original or derived work requires permission of course staff.
- */
-package twitter;
-
-import java.util.List;
-import java.util.ArrayList;
-import java.util.Arrays;
-
-/**
- * Filter consists of methods that filter a list of tweets for those matching a
- * condition.
- * 
- * DO NOT change the method signatures and specifications of these methods, but
- * you should implement their method bodies, and you may add new public or
- * private methods or classes if you like.
- */
-public class Filter {
-
-    /**
-     * Find tweets written by a particular user.
-     * 
-     * @param tweets
-     *            a list of tweets with distinct ids, not modified by this
-     *            method.
-     * @param username
-     *            Twitter username, required to be a valid Twitter username as
-     *            defined by Tweet.getAuthor()'s spec.
-     * @return all and only the tweets in the list whose author is username, in
-     *         the same order as in the input list.
-     */
-    public static List<Tweet> writtenBy(List<Tweet> tweets, String username) {
-        
-        List<Tweet> filteredTweets = new ArrayList<Tweet>();
-        
-        for (int i = 0; i < tweets.size(); i++){
-            
-            if(tweets.get(i).getAuthor().toLowerCase().equals(username.toLowerCase())) {
-                
-                filteredTweets.add(tweets.get(i));
-
-                
-            } else {
-                continue;
-            }
-        }
-        
-        return filteredTweets;
-        
-    }
-
-    /**
-     * Find tweets that were sent during a particular timespan.
-     * 
-     * @param tweets
-     *            a list of tweets with distinct ids, not modified by this
-     *            method.
-     * @param timespan
-     *            timespan
-     * @return all and only the tweets in the list that were sent during the
-     *         timespan, in the same order as in the input list.
-     */
-    public static List<Tweet> inTimespan(List<Tweet> tweets, Timespan timespan) {
-        
-        List<Tweet> filteredTweets = new ArrayList<Tweet>();
-        
-        for (int i = 0; i < tweets.size(); i++){
-            
-            if((tweets.get(i).getTimestamp().isAfter(timespan.getStart())) &&
```

Comment:
I think start and end wouldn't be included here?

Reply:
yes i saw that in your code.... nice one!